cypress/integration/AllAccessibleRecordsPage.feature

```diff
@@ -24,11 +24,3 @@ Feature: the user interacts with the "view all records regardless of OrgUnit"-pa
   Scenario: the working list will hydrate when updated by the URL
     When the user navigates to /#/?programId=VBqh0ynB2wv&all
     Then the working list should be displayed
-
-#  Scenario: The user can edit an event without selecting OrgUnit
```

Comment:
Hey,
Any reason why this scenario was removed? 
Thanks!

Reply:
Good catch. Re-added, fixed and enabled.